clang/test/Driver/darwin-print-library-module-manifest-path.cpp

```diff
@@ -0,0 +1,9 @@
+// Test that -print-library-module-manifest-path finds the correct file.
+//
+// Note this file is currently not available on Apple platforms
```

Comment:
I don't get the intention for the test since there is standard library related things under `%S/Inputs/resource_dir`. Do you mean to search standard library things from the system path? If yes, that is not good and let's move this.

Generally we need a `// UNSUPPORTED:` directive when a lit test can't work in some platforms.

Reply:
I'll remove this test for now. It can be added later when it's known where Apple will store this file.